### PR TITLE
CI

### PR DESCRIPTION
**Pull request by JeremieSamson**

### CODE REVIEW

File: deploy.php [hunk 1]
```diff
@@ -25,8 +25,8 @@
     '.docker',
     '.circleci',
     'docker-compose.yml',
-    'docker-compose.dev.yml',
-    '.env.dist',
+    'docker-compose.override.yml.dist',
+    'docker-compose.ci.yml',
```
Comment (victor-paumier) [inline deploy.php]: what about .env.test?

File: docker-compose.override.yml.dist [hunk 2]
```diff
@@ -0,0 +1,72 @@
+version: '3'
+
+services:
+    php:
+        env_file:
+            - .env.test
```
Comment (victor-paumier) [inline docker-compose.override.yml.dist]: Why do you use test in dev config?

File: .env [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+#TRUSTED_PROXIES=127.0.0.1,127.0.0.2
+#TRUSTED_HOSTS=localhost,example.com
+###< symfony/framework-bundle ###
+
+###> doctrine/doctrine-bundle ###
+DATABASE_URL=mysql://buddysystem:buddysystem@mysql:3306/buddysystem
+###< doctrine/doctrine-bundle ###
+
+###> symfony/swiftmailer-bundle ###
+MAILER_URL=smtp://maildev:25
```
Comment (victor-paumier) [inline .env]: docker-compose.yml do not use maildev

File: docker-compose.ci.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
+version: '3'
+
+services:
+    reverse-proxy:
```
Comment (victor-paumier) [inline docker-compose.ci.yml]: should be also in docker-compose.override?!

File: docker-compose.override.yml.dist (hunk 2 above)
Comment (JeremieSamson) [reply docker-compose.override.yml.dist]: my bad

File: .env (hunk 3 above)
Comment (JeremieSamson) [reply .env]: should we decomment it from the `docker-cpmpose.override.yml` then ?

File: deploy.php (hunk 1 above)
Comment (victor-paumier) [reply deploy.php]: done

File: .env (hunk 3 above)
Comment (victor-paumier) [reply .env]: done

File: docker-compose.override.yml.dist (hunk 2 above)
Comment (victor-paumier) [reply docker-compose.override.yml.dist]: done

File: docker-compose.ci.yml (hunk 4 above)
Comment (victor-paumier) [reply docker-compose.ci.yml]: done

**Pull request merged**